root/layout.tt

```diff
@@ -71,6 +71,17 @@
             </div>
         [%- END -%]
 
+        [%- IF new_edit_notes && new_edit_notes_mtime >
+                (c.req.cookies.new_edit_notes_dismissed_mtime.value || 0) -%]
+            <div class="banner new-edit-notes">
+                <p>
+                    [% l('Notes have been left on some of your edits. Please {uri|click here} to read them.',
```

Comment:
I used `{link|New notes} have been left on some of your edits. Please make sure to read them and respond if necessary.` to sound slightly less commanding, let me know if that can be improved.